command/login_test.go

```diff
@@ -435,16 +438,60 @@ func TestLoginCommand_Run(t *testing.T) {
 		client, closer := testVaultServer(t)
 		defer closer()
 
+		userClient, entityID, methodID, methodName := testhelpers.SetupLoginMFATOTP(t, client)
+		enginePath := testhelpers.RegisterEntityInTOTPEngine(t, client, entityID, methodID)
+
+		runCommand := func(methodIdentifier string) {
+			time.Sleep(21 * time.Second)
```

Comment:
Is there a reason to sleep for 21 seconds specifically? I _think_ this is the time required for the generation of the TOTP code. If so, it might be useful to add a comment as sleeps in tests are sometimes cause for concern but it likely makes sense in this case.

Reply:
Sure, sounds good! Thanks for the feedback.

Reply:
Oh, I see there is one further down too :)

Reply:
SetupLoginMFATOTP is using a period of 20, which I think is where this comes from.  Can we make that configurable and thereby have a quicker test?